packages/core/src/tooltip/Tooltip.css

```diff
@@ -1,39 +1,40 @@
-.uitk-density-touch .uitkTooltip {
-  --icon-top: 6px;
+.uitk-density-touch {
+  --tooltip-icon-top: 6px;
 }
 
-.uitk-density-low .uitkTooltip {
-  --icon-top: 4px;
+.uitk-density-low {
+  --tooltip-icon-top: 4px;
 }
 
-.uitk-density-medium .uitkTooltip {
-  --icon-top: 1px;
+.uitk-density-medium {
+  --tooltip-icon-top: 1px;
 }
 
-.uitk-density-high .uitkTooltip {
-  --icon-top: 1px;
+.uitk-density-high {
+  --tooltip-icon-top: 1px;
 }
 
+/* Styles applied to root component */
 .uitkTooltip {
```

Comment:
Not a change needed in this PR but we should standardise placement of css variables.

Should it go in a separate class, top of the old class, should we use the css var api inline vs creating a private token etc.